.github/workflows/deploy-test.yaml

```diff
@@ -1,9 +1,7 @@
 name: Deploy Test
 
 on:
-  push:
-    tags:
-      - "v[0-9].[0-9]+.[0-9]+-rc*"
+  workflow_dispatch:
```

Comment:
This shouldn't be changing, this makes it a manual workflow and not triggered on tag